package.json

```diff
@@ -21,7 +21,7 @@
     "intl": "1.2.5",
     "jail-monkey": "2.2.0",
     "jsc-android": "241213.2.0",
-    "mattermost-redux": "github:mattermost/mattermost-redux#84748d22aebf07826c6938058e3da0f8d583d6f8",
+    "mattermost-redux": "github:mattermost/mattermost-redux#ed0fa6f1a85c631617b117fb89e7f450811b3465",
```

Comment:
is this commit hash part of the what branch?

Reply:
it is from https://github.com/mattermost/mattermost-redux/tree/mark-as-unread

we have created a `mark-as-unread` feature branch on each repository needed for the feature. Once we merge, we probably have to update this dependency as it will change.

Reply:
ok Just don't forget to do this before the feature branch is merged to master, maybe create a ticket for it?